newrelic-agent/src/main/resources/newrelic.yml

```diff
@@ -284,19 +285,15 @@ common: &default_settings
       # A comma separated list of attribute keys whose values should not be sent to New Relic.
       #exclude:
 
-  # Cross Application Tracing adds request and response headers to
+  # Cross-application tracing adds request and response headers to
   # external calls using supported HTTP libraries to provide better
-  # performance data when calling applications monitored by other New Relic Agents.
-  #
-  # Distributed tracing is replacing cross application tracing as the default
-  # means of tracing between services. To continue using cross application
-  # tracing, enable it with `cross_application_tracer.enabled = true` and
-  # `distributed_tracing.enabled = false`
+  # performance data when calling applications monitored by other New Relic agents.
   cross_application_tracer:
 
-    # Set to true to enable cross application tracing.
-    # Default is false.
-    enabled: false
+    # Set to false to disable cross-application tracing.
+    # Default is true.
+    enabled: true
+
```

Comment:
This isn't accurate.  Distributed tracing is enabled by default now now cross application tracing.

Reply:
I think I may have accidentally pulled in an older version of the config file somehow... I don't recall making this edit.